server/src/main/scala/controllers/application/ExportPortConfigController.scala

```diff
@@ -35,25 +32,24 @@ class ExportPortConfigController @Inject()(cc: ControllerComponents, ctrl: DrtSy
     updatesByTerminalF.map { updatesByTerminal =>
       val eGatesCsv = updatesByTerminal.map { case (k, v) =>
         v.updates.map { updates =>
-          updates.banks.zipWithIndex
-            .map { case (v, i) => s"$k,${SDate(updates.effectiveFrom)},bank-${i + 1}  ${v.openCount}/${v.maxCapacity}" }.mkString("\n")
+          s"$k,${SDate(updates.effectiveFrom).prettyDateTime},${updates.banks.length} bank: ${
+            updates.banks.map { bank => s"${bank.openCount}/${bank.maxCapacity}" }.mkString(" ")
+          }"
         }.mkString("\n")
       }.mkString("\n")
       val eGatesTitle = "E-gates schedule"
       val eGatesHeaders = "Terminal,Effective from,OpenGates per bank"
-      createTitleHeaderAndBody(eGatesTitle, eGatesHeaders, eGatesCsv, true)
+      createTitleHeaderAndBody(eGatesTitle, eGatesHeaders, eGatesCsv)
     }
   }
 
   private def getSlaConfig = {
     val slaUpdates: Future[SlaConfigs] = ctrl.slasActor.ask(GetState).mapTo[SlaConfigs]
     val slaTitle = "Queue SLAs"
-    val slaHeaders = "Effective from,Queue,Minutes"
     slaUpdates.map { sla =>
-      val slaCsv = sla.configs.map { case (eF, vc) =>
-        vc.map { case (queue, v) =>
-          s"${SDate(eF)},$queue,$v"
-        }.mkString("\n")
+      val slaHeaders = s"Effective from,${sla.configs.head._2.map { case (q, _) => s"${Queues.displayName(q)}" }.mkString(",")}"
```

Comment:
We need to make sure the order of the queues in the following rows matches the order in the header row. So rather than using the unsafe `head` method, we should get the port's queues from the airport config and use `Queues.queueOrder` to order them:

```val queuesInOrder = Queues.queueOrder.filter(q => airportConfig.queuesByTerminal.get(thisTerminal).exists(_.contains(q)))```